jitbuilder/optimizer/JBOptimizer.cpp

```diff
@@ -117,11 +117,12 @@ static const OptimizationStrategy JBwarmStrategyOpts[] =
    { OMR::treeSimplification                                                       },
    { OMR::trivialDeadTreeRemoval,                    OMR::IfEnabled                },
 
-   { OMR::inductionVariableAnalysis,                 OMR::IfLoops                  },
-   { OMR::loopCanonicalization,                      OMR::IfLoops                  },
+   { OMR::basicBlockOrdering                                                       }, // clean up block order going into canonicalizer
```

Comment:
Upon further reflection, I suppose there are other benefits to cleaning up block order outside of loop canonicalizer, though perhaps not as noticable...